pool/pool_manager.gno

```diff
@@ -121,15 +132,39 @@ func GetPoolFromPoolPath(poolPath string) *Pool {
 	return pool
 }
 
-// GetPoolPath generates a poolPath from the given token0Path, token1Path, and fee
-// If token0Path comes first in alphabetical order, it returns token0Path:token1Path:fee
-// Otherwise, it returns token1Path:token0Path:fee
+// GetPoolPath generates a poolPath from the given token paths and fee.
+// The poolPath is constructed by joining the token paths and fee with colons.
+//
+// For example, if token0Path is "A" and token1Path is "B" with a fee of 500, the poolPath will be:
+//
+//     "A:B:500"
+//
+// If token0Path is lexicographically greater than token1Path, the paths are swapped before constructing the poolPath.
+//
+//     "B:A:500"
+//
+// It ensures that the token paths are in lexicographic order.
```

Comment:
`lexicographic` vs `alphabetic` ??

Reply:
It doesn't matter. In the context of sorting strings made up of characters, lexicographical order and alphabetical order are the same thing. Just personal prefer.